coderedcms/project_template/pro/website/models.py

```diff
@@ -221,9 +221,17 @@ class Meta:
         ],
         use_json_field=True,
     )
+    site = models.ForeignKey(
+        "wagtailcore.Site",
+        on_delete=models.CASCADE,
```

Comment:
I think SET_NULL would be preferable, in the event that you wanted to switch this navbar to a different site and deleted the site first. Similar to how the root page works.